R/XcmsExperiment-functions.R

```diff
@@ -1169,6 +1209,70 @@ XcmsExperiment <- function() {
     as(MsExperiment(), "XcmsExperiment")
 }
 
+#' Convert a XCMSnExp to a XcmsExperiment.
+#'
+#' @noRd
+.xcms_n_exp_to_xcms_experiment <- function(from) {
```

Comment:
Unusual (to me) way of converting CamelCase class name to (part of) function name. As it is internal, I am just curious.

Reply:
yes, indeed, true. was not paying attention actually. I tend to write internal function names now always in snake_case - for no particular reason except that I find it a bit more readable